fdbserver/ClusterController.actor.cpp

```diff
@@ -1998,7 +2000,15 @@ ACTOR Future<Void> updateDatacenterVersionDifference(ClusterControllerData* self
 				bool oldDifferenceTooLarge = !self->versionDifferenceUpdated ||
 				                             self->datacenterVersionDifference >= SERVER_KNOBS->MAX_VERSION_DIFFERENCE;
 				self->versionDifferenceUpdated = true;
-				self->datacenterVersionDifference = primaryMetrics.get().v - remoteMetrics.get().v;
+				Version logServerVersionDifference = primaryMetrics.get().v - remoteMetrics.get().v;
+				Version storageServerVersionDifference =
+				    (ssVersionLagReply.get().maxPrimarySSVersion > 0 && ssVersionLagReply.get().maxRemoteSSVersion > 0)
+				        ? (ssVersionLagReply.get().maxPrimarySSVersion - ssVersionLagReply.get().maxRemoteSSVersion)
+				        : 0;
+				self->datacenterVersionDifference =
```

Comment:
Do we want to capture these individually in ClusterController and include them in status json output?

Reply:
Also, do we want to introduce a configuration option to consider storage server version difference here (as mentioned in the radar)?

Reply:
Done.